kf_lib_data_ingest/common/misc.py

```diff
@@ -116,6 +117,16 @@ def numeric_to_str(val, replace_na=False, na=None):
 
     :return: new representation of `val`
     """
+    if isinstance(val, list):
+        # make hashable without changing style or losing comparability
+        return str(sorted(
+            convert_to_str(v) for v in val
+        ))
+    if isinstance(val, dict):
+        # make hashable without changing style or losing comparability
+        return str(dict(sorted(
```

Comment:
Same comment as above since Python 3.7 maintains order of keys in dicts

Reply:
but when the dict is converted to a string, you can't then compare equal dicts that were created with their keys in different order

Reply:
Ok I see